DirectEmployers/MyJobs
# Fixed console errors

## What's this PR do?
Fix bugs that were causing console errors.
## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
1. Add a CBO by clicking on "+ New CBO"
2. Fill the CBO name field and click Next.
3. On the Contact form, Click the Back button and Press Okay on the prompt
4. Press the browser back button
5. Click anywhere in the page.
Expected results:
  No console errors.

## Any background context you want to provide?
